keycloak-saml-demo/saml-demo/src/main/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+saml2:
+  rp:
+    signing:
+      # openssl req -newkey rsa:2048 -nodes -keyout rp-key.key -x509 -days 365 -out rp-certificate.crt
+      cert-location: signing/rp-certificate.crt
+      key-location: signing/rp-key.key
+  ap:
+    metadata:
+      location: https://common-logon-dev.hlth.gov.bc.ca/auth/realms/moh_applications/protocol/saml/descriptor
+    signing-cert: #paste Keycloak public certificate from SAML descriptor metadata here
```

Comment:
It's odd that it can't just read the signing-cert from `saml2.ap.metadata.location`. Seems redundant.

We could hardcode `signing-cert` like we hardcoded the `location`, it's public anyhow, but then it would be flagged by the scanning tools as a possible "leaked secret."

Reply:
Good point. I think the other version of code snippet I was trying, was able to fetch cert value from `saml2.ap.metadata.location` but I had trouble with changing the registration ID (client ID in Keycloak) so decided to go with this.